### PR TITLE
v2: Updated hosts and input JSON file

### PR DESCRIPTION
**Pull request by suhanselvan**
Changes in this PR -
-> Added OS-independent hosts group in Ansible inventory file
-> Updated input JSON
-> Added components JSON file

### CODE REVIEW

File: deploy/v2/input.json [hunk 1]
```diff
@@ -1,230 +1,230 @@
...
-  "infrastructure": {
-    "region": "",
-    "resource_group": {
-      "is_existing": "",
-      "arm_id": "",
-      "name": ""
-    },
-    "vnets": {
-      "management": {
+    "infrastructure": {
```
Comment (rukawata) [inline deploy/v2/input.json]: The indention is not consistent- sometimes it's two spaces, sometimes four spaces. Please correct that.

Comment (suhanselvan) [reply deploy/v2/input.json]: Actually it is consistent. For some reason Github is mixing up the line numbers while showing changes. If you look at the actual file it looks fine.

Comment (suhanselvan) [reply deploy/v2/input.json]: Okay. I do notice some inconsistencies. I'll fix them.

Comment (suhanselvan) [reply deploy/v2/input.json]: Fixed.

**Pull request merged**